open-y-subprojects/openy_node_alert
# fix: Count length of alerts with markup

Issue:
Alerts with markup in the description do not get hidden if they are longer than 150 characters.

To test:
- Create an Alert
- Add bold/italics/etc to the Description field and ensure it's longer than 150 characters
- Publish the Alert
- Observe the alert description is properly hidden on mobile

![SCR-20240410-midi](https://github.com/open-y-subprojects/openy_node_alert/assets/238201/5376e8f6-816f-4a17-8b3d-c5ee8ba55f68)
